src/PrestaShopBundle/Form/Admin/Sell/Address/CustomerAddressType.php

```diff
@@ -288,51 +316,54 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ]),
                 ],
             ])
-            ->add('city', TextType::class, [
+            ->add('postcode', TextType::class, [
                 'required' => true,
+                'label' => $this->translator->trans('Zip/Postal code', [], 'Admin.Global'),
+                'empty_data' => '',
                 'constraints' => [
-                    new NotBlank([
-                        'message' => $this->translator->trans(
-                            'This field cannot be empty', [], 'Admin.Notifications.Error'
-                        ),
+                    new AddressZipCode([
+                        'id_country' => $countryId,
+                        'required' => false,
                     ]),
                     new CleanHtml(),
                     new TypedRegex([
-                        'type' => TypedRegex::TYPE_CITY_NAME,
+                        'type' => TypedRegex::TYPE_POST_CODE,
                     ]),
                     new Length([
-                        'max' => AddressConstraint::MAX_CITY_LENGTH,
+                        'max' => AddressConstraint::MAX_POSTCODE_LENGTH,
                         'maxMessage' => $this->translator->trans(
                             'This field cannot be longer than %limit% characters',
-                            ['%limit%' => AddressConstraint::MAX_CITY_LENGTH],
+                            ['%limit%' => AddressConstraint::MAX_POSTCODE_LENGTH],
                             'Admin.Notifications.Error'
                         ),
                     ]),
                 ],
             ])
-            ->add('postcode', TextType::class, [
-                'required' => false,
-                'empty_data' => '',
+            ->add('city', TextType::class, [
+                'label' => $this->translator->trans('City', [], 'Admin.Global'),
+                'required' => true,
                 'constraints' => [
-                    new AddressZipCode([
-                        'id_country' => $countryId,
-                        'required' => false,
+                    new NotBlank([
+                        'message' => $this->translator->trans(
+                            'This field cannot be empty', [], 'Admin.Notifications.Error'
```

Comment:
Or _This field is required_ which might be less negative to read for the user.

Reply:
```suggestion
                            'This field is required', [], 'Admin.Notifications.Error'
```

I agree!